seqr/views/apis/anvil_workspace_api.py

```diff
@@ -236,7 +248,95 @@ def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
         project_name=project.name,
         sample_type=sample_type,
         genome_version=GENOME_VERSION_LOOKUP.get(project.genome_version),
+        dag_name = "seqr_vcf_to_es_{anvil_type}_v{version}".format(anvil_type=sample_type, version=DAG_VERSION),
         dag=json.dumps(pipeline_dag, indent=4),
     )
 
     safe_post_to_slack(SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, message_content)
+
+def _trigger_anvil_dags(project, data_path, user_input):
+    genome_test_type = 'AnVIL_{sample_type}'.format(sample_type=user_input['sampleType'])
+
+    anvil_variables = _get_dag_variables(genome_test_type)
+    updated_anvil_variables = {
+        "active_projects": [project.guid],
+        "es_enable_export": 'true',
```

Comment:
I though there was away to set this for a given dag and not need to set it on every dag run. Lets do that instead